docs/user/merging-metadata.md

```diff
@@ -0,0 +1,93 @@
+# Merging metadata {#merging-metadata}
+
+## About merging metadata {#about-merging-metadata}
+
+## How to Merge Indicators {#indicators-merge}
+
+The indicator merge feature lets you combine multiple source indicators into a single target indicator. Follow these steps to merge indicators:
+
+1. **Go to the Indicators List:**
+   Click on **Indicators -> Indicator** in the left-hand side (LHS) menu.
+
+2. **Select Source Indicators:**
+   Check the boxes next to the indicators you want to merge.
+
+3. **Click the Merge Button:**
```

Comment:
Should we add a note that you need the merge authority to do this? Either here on in the intro-text above? Or maybe both? 

```
Note: This button will not appear if you do not have the `Merge indicators` authority. 
```